changelogs/api-management-service.md

```diff
@@ -1,5 +1,20 @@
 # Azure API Management service changelog
 
+## Release - API Management service: December 2022
+
+A regular Azure API Management service update was started on December 8, 2022. This release will continue to roll out through January 2023.
+
+### New features, fixes, and improvements
+
+1. The `log-to-eventhub` policy now supports securing connections to Azure Event Hub with managed identity.
```

Comment:
Can we add the control plane events that we have discussed please'?

Reply:
For example, "Provide control-plane events in Azure Event Grid for self-hosted gateway"

Reply:
Let's leave it out